crates/sargon-uniffi/src/system/sargon_os/entities_controlled_by_factor_source/support/accessibility/device.rs

```diff
@@ -0,0 +1,15 @@
+use crate::prelude::*;
+use sargon::DeviceFactorSourceAccessibility as InternalDeviceFactorSourceAccessibility;
+
+/// A struct representing the accessibility of a device factor source.
```

Comment:
I think `accessibility` is the wrong term. I think you mean `existence` ? Accessibility make it sound like [keychain *accessibility*](https://developer.apple.com/documentation/security/restricting-keychain-item-accessibility), which is a specific keychain **configuration**.

Reply:
Right we have `is_mnemonic_marked_as_backed_up`, so what about: `ClaimedAndActualExistenceOfDeviceFactorSource`

Reply:
I didn't name based on keychain accesibility, but based on the definition of the word itself:
> the quality of being able to be reached or entered.

Reply:
Yeah I understood that :) my point is it is a loaded term, especially since we are touching something very Keychain related